deploy/helm/templates/deployment.yaml

```diff
@@ -0,0 +1,89 @@
+      {{- end }}
+      {{- if .Values.affinity }}
+      affinity: {{- include "grafana-operator.tplValue" (dict "value" .Values.affinity "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.tolerations}}
+      tolerations: {{- include "grafana-operator.tplValue" (dict "value" .Values.tolerations "context" $) | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "grafana-operator.serviceAccountName" . }}
+      containers:
+        - name: grafana-operator
+          image: {{ include "grafana-operator.image" . }}
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+          - containerPort: {{ .Values.service.port }}
+            name: metrics
+          command:
+          - grafana-operator
+          args:
+          - -grafana-image={{ include "grafana-operator.grafana.baseImage" . }}
+          - -grafana-image-tag={{ .Values.grafana.image.tag | quote}}
+          - -grafana-plugins-init-container-image={{ include "grafana-operator.pluginInit.baseImage" . }}
+          - -grafana-plugins-init-container-tag={{ .Values.grafanaPluginInit.image.tag | quote}}
+          {{- if (and .Values.args.scanAllNamespaces (not .Values.args.scanNamespaces)) }}
+          - --scan-all=True
+          {{- else if .Values.args.scanNamespaces }}
+          - --namespaces={{ include "grafana-operator.joinListWithComma" .Values.args.scanNamespaces }}
+          {{- end }}
+        {{- if .Values.resources }}
+          resources: {{- toYaml .Values.resources | nindent 12 }}
+        {{- end }}
+        {{- if .Values.readinessProbe.enabled }}
+          readinessProbe:
+            exec:
+              command:
+                - stat
+                - /tmp/operator-sdk-ready
+            initialDelaySeconds: {{ .Values.readinessProbe.initialDelaySeconds }}
+            periodSeconds: {{ .Values.readinessProbe.periodSeconds }}
+            timeoutSeconds: {{ .Values.readinessProbe.timeoutSeconds }}
+            successThreshold: {{ .Values.readinessProbe.successThreshold }}
+            failureThreshold: {{ .Values.readinessProbe.failureThreshold }}
+        {{- end }}
+        {{- if .Values.livenessProbe.enabled }}
+          livenessProbe:
+            httpGet:
+              path: /metrics
+              port: metrics
+            initialDelaySeconds: {{ .Values.livenessProbe.initialDelaySeconds }}
+            periodSeconds: {{ .Values.livenessProbe.periodSeconds }}
+            timeoutSeconds: {{ .Values.livenessProbe.timeoutSeconds }}
+            successThreshold: {{ .Values.livenessProbe.successThreshold }}
+            failureThreshold: {{ .Values.livenessProbe.failureThreshold }}
+        {{- end }}
+          env:
+            - name: TEMPLATE_PATH
+              value: /usr/local/bin/templates
+            - name: WATCH_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
```

Comment:
@Cellebyte can we have this in values.yaml so that it can be optional for those who want to use it both a cluster scope and namespace scope. For clusterscope this block fits
```
- name: WATCH_NAMESPACE
  value: ""
```

Reply:
implemented